packages/cli/src/index.ts

```diff
@@ -0,0 +1,14 @@
+const inquirer = require('inquirer');
+
+inquirer
+  .prompt([
+    {
+      name: 'helloWorldName',
+      message: 'What is your name?',
+    },
+  ])
+  // eslint-disable-next-line promise/always-return
+  .then((answers) => {
+    // eslint-disable-next-line no-console
+    console.info('Hello ', answers.helloWorldName);
+  });
```

Comment:
```suggestion
(async () => {
  const answers = inquirer
  .prompt([
    {
      name: 'helloWorldName',
      message: 'What is your name?',
    },
  ]);
  
  console.info('Hello ', answers.helloWorldName);
})();
```

This can be converted to async/await with a [self invoked function](https://scriptverse.academy/tutorials/js-self-invoking-functions.html#:~:text=A%20self%2Dinvoking%20(also%20called,invoked%20immediately%20after%20its%20definition.&text=A%20self%2Dinvoking%20function%20can,be%20passed%20as%20a%20parameter.).

And from node version 13 we have [top-level await](https://opensource.com/article/20/11/top-level-await-javascript) aswell :)

Reply:
No problem I will definitely add it. Currently, it is only a hello world project 🙃